scipy/optimize/_qap.py

```diff
@@ -43,97 +41,100 @@ def quadratic_assignment(
     Parameters
     ----------
     cost_matrix : 2d-array, square, non-negative
-        A square adjacency matrix. In this implementation, :math:`A` =
-        `cost-matrix` in the objective function above.
+        The square adjacency matrix :math:`A` in the objective function above.
 
     dist_matrix : 2d-array, square, non-negative
-        A square adjacency matrix.  In this implementation, :math:`B` =
-        `dist-matrix` in the objective function above.
-
-    partial_match : 2d-array, optional, (default = None)
-        Allows the user apply a seed, fixing part of the matching between
-        the two adjacency matrices.
-        For column 1, each entry is an index of a node in `cost_matrix`.
-        For column 2, each entry is an index of a node in `dist_matrix`.
-        The elements of ``seed[:, 0]`` and ``seed[:, 1]`` are vertices
-        which are known to be matched, that is, ``seed[i, 0]`` is matched to
-        vertex ``seed[i, 1]``. Array shape ``(m , 2)`` where ``m <= number of
-        nodes``.
+        The square adjacency matrix :math:`B` in the objective function above.
+
+    partial_match : 2d-array of integers, optional, (default = None)
+        Allows the user to fix part of the matching between the two adjacency
+        matrices. In the literature, a partial match is also known as a
+        "seed" [2]_.
+
+        Each row of `partial_match` specifies the indices of a pair of
+        corresponding nodes, that is, vertex ``partial_match[i, 0]`` is
+        matched to vertex ``partial_match[i, 1]``. Accordingly,
```

Comment:
Oops! This is what I meant.
```suggestion
        corresponding nodes, that is, node``partial_match[i, 0]`` of `cost_matrix` is
        matched to node``partial_match[i, 1]`` of `dist_matrix`. Accordingly,
```